docs/backend/guides/guide-to-history-and-audit-logging.md

```diff
@@ -0,0 +1,38 @@
+---
+sidebar_position: 23
+---
+
+# History and Audit Logging Guide
+
+## Tracking Event History from End to End
+- [Database Triggers](#database-triggers)
+- [AppContext](#appcontext-in-context)
+- [Fetcher](#move-history-fetcher)
+- [Frontend](#frontend-event-templates)
+
+### Database Triggers
+Move history events are created by [Event Triggers](docs/backend/guides/how-to/add-an-event-trigger/) and are collected in a single :lock:[move history table](https://dp3.atlassian.net/wiki/spaces/MT/pages/1634992192/PO9+Technical+Implementation) which captures what event took place, what data changed, and which user initiated the event. 
```

Comment:
**suggestion**: These internal links weren't working on my machine, I think they might need to start with a slash

```suggestion
Move history events are created by [Event Triggers](/docs/backend/guides/how-to/add-an-event-trigger/) and are collected in a single :lock:[move history table](https://dp3.atlassian.net/wiki/spaces/MT/pages/1634992192/PO9+Technical+Implementation) which captures what event took place, what data changed, and which user initiated the event.
```

Reply:
Ahhhh thanks for catching that. I pushed a fix.